index.js

```diff
@@ -40,6 +41,17 @@ app.post('/like', (request, response) => {
   });
 });
 
+app.get('/api/search/*', (req, res) => {
+  var results = userStore.searchUsers(req.params[0]);
+
+  console.log(chalk.green('RESULTS ARE:'));
+  console.log(results);
+  
+  res.json(results);
+});
+
+
+
 app.listen(3000, () => {
   console.log('Web Server is running on port 3000');
 });
```

Comment:
it is a short hand in express to set HTTP Response 'Content-Type' header to 'application/json' and send the json in the response body by JSON.stringifying the JavaScript object: http://expressjs.com/en/4x/api.html#res.json